code/game/gamemodes/blob/blobs/shield.dm

```diff
@@ -2,8 +2,9 @@
 	name = "strong blob"
 	icon = 'icons/mob/blob.dmi'
 	icon_state = "blob_idle"
-	desc = "Some blob creature thingy"

```

Comment:
I'm killing the fuck out of it, because I've hated it for as long as I've known it was a thing.